Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+use num_bigint::{BigUint, ParseBigIntError};
+use std::io;
+use std::str::FromStr;
+
+// Number of days in a year
+pub const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+// Takes a year as an argument and returns the equivalent in days
```

Comment:
You could turn this into a docstring to generate docs with `cargo doc` and include examples as doctests, run during `cargo test`. Check my pr and these:
https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html
https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#documentation-comments-as-tests